root/static/scripts/release-editor/duplicates.js

```diff
@@ -169,10 +173,13 @@ function combinedMediumFormatName(mediums) {
     var formats = pluck(_(mediums), "format");
     var formatCounts = formats.countBy(_.identity);
 
-    return formats.uniq().map(function (format) {
+    return formats
+    .uniq()
```

Comment:
All the method calls in this chain here and below should be indented another 4 spaces